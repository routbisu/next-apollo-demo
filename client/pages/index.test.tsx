/**
 * @jest-environment jsdom
 */

import React from 'react'
import { shallow } from 'enzyme'
import HomePage from '@pages/index'

describe('HomePage', () => {
  it('should render the heading', async () => {
    const wrapper = shallow(<HomePage />)

    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find({ 'data-testid': 'heading' }).text()).toEqual(
      'Registered Users'
    )
  })
})
