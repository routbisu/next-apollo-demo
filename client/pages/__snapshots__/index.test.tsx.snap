// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render the heading 1`] = `
<PageContainer>
  <Heading__PageHeading
    data-testid="heading"
  >
    Registered Users
  </Heading__PageHeading>
  <Grid>
    <Loader />
  </Grid>
</PageContainer>
`;
